core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -1005,12 +1006,27 @@ class KafkaZkClient private (zooKeeperClient: ZooKeeperClient, isSecure: Boolean
   }
 
   /**
-   * Creates Acl change notification message
-   * @param resource resource name
+    * Creates colon separated Acl change notification message.
+    *
+    * Kafka 2.0 saw the format of the ACL change event change from a 'colon separated' string, to a JSON message.
```

Comment:
The comment needs to be adjusted since we are no longer using JSON.

Reply:
Good catch - done.